src/transform/EditTransform.js

```diff
@@ -0,0 +1,34 @@
+import './EditTransform.css'
+
+const CLASS = {
+  CONTAINER: 'pagelib_edit_section_link_container',
+  LINK: 'pagelib_edit_section_link',
+  PROTECTION: { UNPROTECTED: '', PROTECTED: 'page-protected', FORBIDDEN: 'no-editing' }
+}
+
+const DATA_ATTRIBUTE = { SECTION_INDEX: 'data-id', ACTION: 'data-action' }
+const ACTION_EDIT_SECTION = 'edit_section'
+
+/**
+ * @param {!Document} document
+ * @param {!number} index The zero-based index of the section.
+ * @return {!HTMLSpanElement}
+ */
+const newEditSectionLink = (document, index) => {
+  const container = document.createElement('span')
```

Comment:
Something about calling it a link but returning a span seemed odd... I liked `icon` because it conveyed the button-ness... though I also like newEditSection**Button**Span too...